### DISCUSSION TITLE
Port Request for Notepad++

### DISCUSSION THREAD

**Discussion by DepressedOptimist** [daltonmenezes/aura-theme#207] [Port request]
I only have access to Notepad++ at work and would really like to have this theme on it, but I do not have the skill to do it myself.

**Comment by VinMarianni** [comment]
Did someone somehow successfully install theme to npp? Please :)

**Comment by GTFBO** [comment]
Pleasie czy ktoś może pomóc ?

**Comment by daltonmenezes** [comment]
@DepressedOptimist @VinMarianni @GTFBO 

Done! 🎉 https://github.com/daltonmenezes/aura-theme/pull/238
